src/modal/alert-modal.component.ts

```diff
@@ -106,4 +111,9 @@ export class AlertModal extends BaseModal {
 		}
 		this.closeModal();
 	}
+
+	isOverflow() {
+		const element = this.content.nativeElement;
```

Comment:
We should guard this for cases where the view may be delayed ... just a simple `if (!this.content) { return false; }` should be fine 👍